package.json

```diff
@@ -66,13 +66,15 @@
     "graphql-scalars": "^1.14.1",
     "graphql-tag": "^2.12.6",
     "graphql-upload": "^13.0.0",
-    "luxon": "^2.3.0"
+    "luxon": "^2.3.0",
+    "node-fetch": "^2.6.7"
   },
   "devDependencies": {
     "@types/chai": "4.3.0",
     "@types/faker": "5.5.9",
     "@types/graphql-upload": "8.0.7",
     "@types/jest": "27.4.1",
+    "@types/node": "^17.0.21",
```

Comment:
We're on Node 16 in this repo, if the types have to be added to the packages explicitly, shouldn't they be v.16 rather than v.17?

Reply:
updated